app/helpers/locations_helper.rb

```diff
@@ -107,4 +107,15 @@ def is_nested?(inner, outer)
 
     [inner.deactivation_date, inner.reactivation_date].all? { |date| date.between?(outer.deactivation_date, outer.reactivation_date) }
   end
+
+  def formatted_local_authority_timeline(location, field)
+    sorted_linked_authorities = location.linked_local_authorities.sort_by(&:start_date)
+    return sorted_linked_authorities.first[field] if sorted_linked_authorities.count == 1
+
+    sorted_linked_authorities.map { |linked_local_authority|
+      formatted_start_date = linked_local_authority.start_date&.to_formatted_s(:govuk_date)
+      formatted_end_date = linked_local_authority.end_date&.to_formatted_s(:govuk_date)
+      "#{linked_local_authority[field]} (#{formatted_start_date} - #{formatted_end_date || 'present'})"
+    }.join("\n")
+  end
```

Comment:
```suggestion
  def formatted_local_authority_timeline(location, field)
    sorted_linked_authorities = location.linked_local_authorities.sort(:start_date)
    
    return sorted_linked_authorities.first[field] if sorted_linked_authorities.count.one?

    sorted_linked_authorities.map do |linked_local_authority|
      formatted_start_date = linked_local_authority.start_date&.to_formatted_s(:govuk_date)
      formatted_end_date = linked_local_authority.end_date&.to_formatted_s(:govuk_date) || "present"
      "#{linked_local_authority[field]} (#{formatted_start_date} - #{formatted_end_date})"
    end.join("\n")
  end
```

https://github.com/rubocop/ruby-style-guide#single-line-blocks-delimiters

However! I'd challenge whether we need that return statement:

```suggestion
  def formatted_local_authority_timeline(location, field)
    sorted_linked_authorities = location.linked_local_authorities.sort(:start_date)

    sorted_linked_authorities.map do |linked_local_authority|
      formatted_start_date = linked_local_authority.start_date&.to_formatted_s(:govuk_date)
      formatted_end_date = linked_local_authority.end_date&.to_formatted_s(:govuk_date) || "present"
      "#{linked_local_authority[field]} (#{formatted_start_date} - #{formatted_end_date})"
    end.join("\n")
  end
```

Thoughts?

Reply:
Hm, yes to the first comment, not too sure about removing the return. This will impact a very small number of LAs, so maybe we shouldn't display the dates by default

Reply:
(Note the AR sorted vs Enumerator.)

Reply:
That's okay tho methinks, cause there'd be at most 2 records there anyway